api/client/samples/crop_models/requirements.txt

```diff
@@ -1,7 +1,11 @@
-pandas
-matplotlib
+descartes
```

Comment:
descartes is a dependency of shapely, which itself is a dependency of geopandas. does that mean we dont need to list it in requirements explicitly?

Reply:
good question... I think it was added in the TRS notebook since that didn't need geopandas. Doesn't hurt to leave be more explicit here.